static/app/views/settings/organizationAuditLog/auditLogList.tsx

```diff
@@ -270,53 +271,59 @@ function AuditLogList({
 
   return (
     <div>
-      <SettingsPageHeader title={t('Audit Log')} action={action} />
-      <PanelTable
-        headers={[t('Member'), t('Action'), t('IP'), t('Time')]}
-        isEmpty={!hasEntries && entries?.length === 0}
-        emptyMessage={t('No audit entries available')}
-        isLoading={isLoading}
-      >
-        {(entries ?? []).map(entry => {
-          if (!entry) {
-            return null;
-          }
-          return (
-            <Fragment key={entry.id}>
-              <UserInfo>
-                <div>{getAvatarDisplay(entry.actor)}</div>
-                <NameContainer>
-                  {addUsernameDisplay(entry.actor)}
-                  <AuditNote entry={entry} orgSlug={organization.slug} />
-                </NameContainer>
-              </UserInfo>
-              <FlexCenter>
-                <MonoDetail>{getTypeDisplay(entry.event)}</MonoDetail>
-              </FlexCenter>
-              <FlexCenter>
-                {entry.ipAddress && (
-                  <IpAddressOverflow>
-                    <Tooltip
-                      title={entry.ipAddress}
-                      disabled={entry.ipAddress.length <= ipv4Length}
-                    >
-                      <MonoDetail>{entry.ipAddress}</MonoDetail>
-                    </Tooltip>
-                  </IpAddressOverflow>
-                )}
-              </FlexCenter>
-              <TimestampInfo>
-                <DateTime dateOnly date={entry.dateCreated} />
-                <DateTime
-                  timeOnly
-                  format={is24Hours ? 'HH:mm zz' : 'LT zz'}
-                  date={entry.dateCreated}
-                />
-              </TimestampInfo>
-            </Fragment>
-          );
-        })}
-      </PanelTable>
+      {!organization.access ? (
```

Comment:
we want to check if you have sufficient permission to hit the endpoint and actually see something (not 403).

to check for `org:write` scope you can do `organization.access?.includes('org:write')`. if somebody doesn't have `org:write` we want to show the `PermissionAlert`